uwblueprint/planet-read
# Insert test data into database through python

## Notion ticket link

[seed db from python](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=4a47edc475b34bb99b25ff1fae2a5507&p=505cfd575fba45d0a5710682e1de18ed)



## Implementation description
* Same as https://github.com/uwblueprint/planet-read/pull/81, but moved the `insert_test_data.py` file out of `python/app` and put it in a new `python/tools` directory
* Run this seed script on heroku deployments



## Steps to test
1. Rebuild containers: `docker-compose down --volumes` and `docker-compose up --build`
2. Ensure database tables are created
3. Run `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"` to ensure the tables are created
4. Run `docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data"`
5. Check that database is seeded: `docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; SELECT * from comments;"`


## What should reviewers focus on?
* Is it okay to have `insert_test_data.py` outside of the app folder itself?


## Checklist
